lib/commands/history.js

```diff
@@ -4,23 +4,26 @@ const math = require('../utils/math');
 const formatters = require('../utils/formatters');
 const cli = require('../utils/cli');
 
-async function history({ cwd, host, user, repo, branch = '*', result = '*', json = false }) {
+async function history({ cwd, host, user, repo, threshold, branch = '*', result = '*', json = false }) {
   let history = await builds.getHistory(cwd, host, user, repo, { branch, result });
   let durations = history.map(build => build.duration);
   let { min, max } = math.getMinMax(durations);
+  if (threshold == undefined) {
```

Comment:
Nitpick, but you should never use the standard equality check for undefined because it'll also be true for null (not an issue in this case, but just for future reference)